project-base/storefront/pages/order/transport-and-payment.tsx

```diff
@@ -25,7 +26,21 @@ const TransportAndPaymentPage: FC<ServerSidePropsType> = () => {
 export const getServerSideProps = getServerSidePropsWrapper(
     ({ redisClient, domainConfig, t }) =>
         async (context) =>
-            initServerSideProps({ context, redisClient, domainConfig, t }),
+            initServerSideProps({
```

Comment:
It would be nice if you've added the variable type here

```suggestion
            initServerSideProps<TypeAdvertsQueryVariables>({
```